app/src/main/java/com/conupods/OutdoorMaps/BuildingDataMap.java

```diff
@@ -12,6 +12,7 @@
 import java.io.InputStreamReader;
 import java.nio.charset.StandardCharsets;
 import java.util.HashMap;
```

Comment:
no longer needed

Reply:
hashmap

Reply:
No its still needed, mData is declared as a util.Map but still initialized as a util.HashMap, see line 30

Reply:
nevermind the newer version of master doest use it anymore. It has been removed here too.